### PR TITLE
IOContext multiline attribute is deprecated

### PR DESCRIPTION
**Pull request by vtjnash**

### CODE REVIEW

File: src/DocChecks.jl [hunk 1]
```diff
@@ -287,7 +287,7 @@ end
 if VERSION < v"0.5.0-dev+4305"
     text_display(buf) = TextDisplay(buf)
 else
-    text_display(buf) = TextDisplay(IOContext(buf, multiline = true, limit = true))
+    text_display(buf) = TextDisplay(IOContext(buf, :limit => true))
```
Comment (tkelman) [inline src/DocChecks.jl]: does this work correctly in all versions 0.5.0-dev+4305 and higher?

Comment (vtjnash) [inline src/DocChecks.jl]: Yes. I verified this was never part of a released version of Julia.
```
commit ce231746d0413fb6f3acc23c41abe4240837d858
Author: Jeff Bezanson <jeff.bezanson@gmail.com>
Date:   Fri May 13 16:23:11 2016 -0400

    use IOContext more consistently. part of #14052
    
    We had several `writemime` definitions whose purpose was to control how
    objects should be printed in the REPL. Instead this introduces a
    `:multiline` IOContext flag.
```
```
commit 256a29627b79690eb5e43bc90ffe78c4efd13005
Author: Jameson Nash <vtjnash@gmail.com>
Date:   Sat Jun 25 12:39:48 2016 -0400

    remove io :multiline attribute
```

Comment (vtjnash) [inline src/DocChecks.jl]: Yes. I verified this was never part of a released version of Julia.
```
commit ce231746d0413fb6f3acc23c41abe4240837d858
Author: Jeff Bezanson <jeff.bezanson@gmail.com>
Date:   Fri May 13 16:23:11 2016 -0400

    use IOContext more consistently. part of #14052
    
    We had several `writemime` definitions whose purpose was to control how
    objects should be printed in the REPL. Instead this introduces a
    `:multiline` IOContext flag.
```
```
commit 256a29627b79690eb5e43bc90ffe78c4efd13005
Author: Jameson Nash <vtjnash@gmail.com>
Date:   Sat Jun 25 12:39:48 2016 -0400

    remove io :multiline attribute
```

**Comment by mortenpi** [comment]
Thanks!

**Pull request merged**